src/runtime_src/core/tools/common/OptionOptions.cpp

```diff
@@ -48,3 +50,22 @@ OptionOptions::printHelp() const
                                m_positionalOptions, m_globalOptions);
 }
 
+std::vector<std::string> 
+OptionOptions::process_arguments( boost::program_options::variables_map& vm,
+                                  const SubCmdOptions& _options,

```

Comment:
Could we drop the underscore in the variable name.  For new code going forward it is preferred that this practice isn't done any more.
```suggestion
                                  const SubCmdOptions& options,

```